source/microbit/microbitimage.cpp

```diff
@@ -440,16 +440,6 @@ mp_obj_t microbit_image_invert(mp_obj_t self_in) {
 }
 MP_DEFINE_CONST_FUN_OBJ_1(microbit_image_invert_obj, microbit_image_invert);
 
-mp_obj_t microbit_image_slice_func(mp_uint_t n_args, const mp_obj_t *args) {
```

Comment:
There is still a lot of other image slice functionality that can be deleted (eg microbit_image_slice function).